CROSS-CUTTING-CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+# Creating a development environment
+
+When cross cutting changes are needed, it is best to work in a personal branch, but would need to replicate this to each submodule impacted.
+
+```
+$ git checkout -b CASSANDRA-<number>
```

Comment:
It's assumed we're starting in the project root here?

Reply:
yeah, that's how I wrote it... some things work just fine but never bothered tested if you do this in sub-directories